src/risknet/config/conf_template.yaml

```diff
@@ -0,0 +1,18 @@
+# COPY EVERYTHING BELOW INTO conf.yaml (Create if doesn't exist)
+general:
+  app_name: risknet
+  log_bucket: gs://albell-logs/
+  dask_cluster_type: local
+  gcp_dask_workers: 4
+data:
+  titanic_root: gs://albell-test/
+  fm_root : "YOUR_FM_ROOT" #TODO: Add your fm_root
```

Comment:
In the future, we plan for everyone to use DSMLP. Can we set `fm_root = "../../../../../teams/a15/data/"` now? Or (maybe better) can we add `fm_root = ../../../../../teams/a15/data/` as a comment here?

Reply:
yes, you can add it as a path if that's the working path. Alternatively you can hard code it in the config module's __init__.py or have a constants.py module somewhere.